frappe/auth.py

```diff
@@ -621,7 +619,7 @@ def validate_oauth(authorization_header):
 			frappe.set_user(frappe.db.get_value("OAuth Bearer Token", token, "user"))
 			frappe.local.form_dict = form_dict
 	except AttributeError:
-		raise frappe.AuthenticationError
+		pass
```

Comment:
Why not raise error?

Reply:
- If auth_hook validates header `"Authorization: Bearer {j.w.t}"` and sets the user from it from third party provider
- Then again it finds Bearer in mariadb and fails when it doesn't find it.

There is no silent failure because of this:

 https://github.com/frappe/frappe/blob/443d508669015ef8c6362f402cfb8170bf52e6af/frappe/auth.py#L577-L580